content/docs/command-reference/exp/pull.md

```diff
@@ -0,0 +1,65 @@
+# exp pull
+
+Download a single [experiment](/doc/command-reference/exp) from a Git remote, and
+its data from a `dvc remote`.
```

Comment:
Why only a single experiment?

Reply:
It was just written that way to start with. Internally we can already push/pull multiple experiments at once, but we aren't really sure how users will want to use the experiment sharing via git remotes in the long run. Once we get more user feedback after 2.0 release we can update the CLI commands as needed.

Reply:
OK cool.

> we aren't really sure how users will want to use the experiment sharing via git remotes

Maybe approximate Git? Push all the experiments derived from the current `HEAD` by default (make `experiment` an optional arg, could even be `commit` (revision) instead so you can specify another base commit, or a specific experiment). Just an idea